src/app/suggestion-notifications/qa/project-entry-import-modal/project-entry-import-modal.component.html

```diff
@@ -0,0 +1,71 @@
+<div class="modal-header">
+  <h4 class="modal-title" id="modal-title">{{ (labelPrefix + label + '.title') | translate }}</h4>
+  <button type="button" class="close" aria-label="Close button" aria-describedby="modal-title"
+          (click)="close()">
+    <span aria-hidden="true">&times;</span>
+  </button>
+</div>
+<div class="modal-body">
+  <small>{{ (labelPrefix + label + '.publication' | translate) }}</small>
+  <div class="mb-3">
+    <div class="text-truncate">
+      <a target="_blank" href="/items/{{(externalSourceEntry.event.target|async)?.payload?.id}}">
```

Comment:
This `href` should be replaced with `[routerLink]`, otherwise this link may break if you deploy the UI on a subpath.